snapcraft/internal/project_loader/_extensions/kde_neon.py

```diff
@@ -30,8 +30,22 @@ class ExtensionImpl(Extension):
     This extension makes it easy to assemble KDE based applications
     using the Neon stack.
 
-    The version of Neon used is dependent on the base used to create
-    the snap.
+    It configures each application with the following plugs:
+
+    \b
+    - Common Icon Themes.
+    - Common Sound Themes
```

Comment:
nit: missing period to match line before & after  (either way, but consistent)

Reply:
Fixed, thanks! The gnome extension also had the same issue, I fixed it there too.